clock/testing/fake_clock_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package clock
+package testing
```

Comment:
we don't have any test for the real clock? I guess it's not that easy to test ...

Reply:
yeah, we are just using golang's time library